modules/programs/neovim.nix

```diff
@@ -198,6 +198,16 @@ in {
         '';
       };
 
+      extraWrapperArgs = mkOption {
+        type = with types; listOf str;
+        default = [ ];
+        example =
+          literalExpression ''[ "--suffix" "PATH" ":" "/path/to/bin" ]'';
```

Comment:
maybe put a working example instead, like `${golsp}/bin/golsp` (make sure to escape `${golsp}` suc htat it doesn't get interpolated.

Reply:
also this example might not be the best because one should use `extraPackages` instead. Take the example from what you want to achieve with this PR, aka pass additionnal variables for the linker and make mason work.
Bonus points if you can test it but that's not mandatory, the implementation is simple and is already tested in nixpkgs.

Reply:
Is it okay to just add items to `tests/modules/programs/neovim/no-init.nix` or `tests/modules/programs/neovim/runtime.nix` for the test?

Reply:
add test to [runtime.nix](https://github.com/nix-community/home-manager/blob/master/tests/modules/programs/neovim/runtime.nix).